### PR TITLE
Switched to main branch

### PR DESCRIPTION
**Pull request by milessabin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,7 +163,7 @@ releases][olderusage] on the shapeless wiki.
 
 ## Building shapeless
 
-shapeless is built with SBT 1.3.10 or later, and its master branch is built with Scala 2.13.2 by default but also
+shapeless is built with SBT 1.3.10 or later, and its main branch is built with Scala 2.13.2 by default but also
 cross-builds for 2.11.12 and 2.12.12.
```
Comment (joroKr21) [inline README.md:167]: I don't know if we need to mention concrete versions, they are always out of date

File: README.md [hunk 2]
```diff
@@ -75,7 +74,7 @@ being [good candidates to take on][goodfirstissue]. No contribution is too small
 ## Using shapeless
 
 Binary release artefacts are published to the [Sonatype OSS Repository Hosting service][sonatype] and synced to Maven
-Central. Snapshots of the master branch are built using [Travis CI][ci] and automatically published to the Sonatype
+Central. Snapshots of the main branch are built using [Travis CI][ci] and automatically published to the Sonatype
```
Comment (joroKr21) [inline README.md:77]:
```suggestion
Central. Snapshots of the main branch are built using GitHub actions and automatically published to the Sonatype
```

File: README.md (hunk 1 above)
Comment (milessabin) [reply README.md:167]: I'll pick that up later.

File: README.md (hunk 2 above)
Comment (milessabin) [reply README.md:77]: Good spot.

**Review by joroKr21 — APPROVED** [review]

**Pull request merged**